awx/main/conf.py

```diff
@@ -825,6 +828,7 @@
     help_text=_('Max number of hosts to allow to be created in a single bulk action'),
     category=_('Bulk Actions'),
     category_slug='bulk',
+    hidden=True,
```

Comment:
why are these other settings being hidden?

Reply:
they are not being shown on the UI anyway, and currently its hard coded for the UI to not show these settings

UI_NEXT want this to be dynamic and be determined by the API response